elasticsearch-transport/lib/elasticsearch/transport/transport/http/manticore/adapter.rb

```diff
@@ -0,0 +1,58 @@
+module Elasticsearch
+  module Transport
+    module Transport
+      module HTTP
+        class Manticore
+          class Adapter
```

Comment:
Can we have a "header" documentation text here, describing the purpose of this class, and how it fits with the Manticore-specific architecture?